docs/private-networks/reference/cli/options.md

```diff
@@ -922,6 +922,48 @@ privacy-url="http://127.0.0.1:8888"
 
 The URL on which the [Tessera node](../../tutorials/privacy/index.md#3-create-tessera-configuration-files) is running.
 
+### `snapsync-bft-enabled`
```

Comment:
do we want to add the version of besu that this becomes available eg (since 24.12.0)